File: src/main/java/com/talenco/lotterysystem/service/GoodService.java
package com.talenco.lotterysystem.service;

import com.talenco.lotterysystem.POJO.Employee;
import com.talenco.lotterysystem.POJO.Good;

import java.util.List;

public interface GoodService {

    List<Good> getAll();

    Good getOne(Long id);

    void insert(Good good) throws Exception;

    void update(Good good);

    void delete(Long[] ids);
}

File: src/main/webapp/js/service/good_service.js
app.service('goodService', function ($http) {

    this.getAll = function () {
        return $http.get('../GoodController/getAll');
    };

    this.getOne = function (gid) {
        return $http.get('../GoodController/getOne?gid=' + gid)
    };

    this.insert = function (entity) {
        return $http.post('../GoodController/insert', entity)
    };

    this.update = function (entity) {
        return $http.post('../GoodController/update', entity)
    };

    this.delete = function (ids) {
        return $http.post('../GoodController/delete?ids=' + ids)
    };
})
